gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/ClusterStatistics.java

```diff
@@ -95,33 +92,28 @@ public void updateClusterStatistics(Map<String, MemberStatistics> memberStatsMap
     long sumAggrMemoryUsed;
     long sumDiskStoreDiskSpace = 0;
 
-    Set<String> hostsList = new HashSet<>();
+    Set<String> hostsListForCPU = new HashSet<>();
```

Comment:
Can you please rename "hostListForCPU" to "hostListForCPUUsage" ?

Reply:
Done.